src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/notification/ClaimReconsiderationUpheldDefendantNotificationHandler.java

```diff
@@ -0,0 +1,118 @@
+            callbackKey(ABOUT_TO_SUBMIT),
+            this::notifyClaimReconsiderationUpheld
+        );
+    }
+
+    @Override
+    public String camundaActivityId(CallbackParams callbackParams) {
+        return TASK_ID;
+    }
+
+    @Override
+    public List<CaseEvent> handledEvents() {
+        return EVENTS;
+    }
+
+    private CallbackResponse notifyClaimReconsiderationUpheld(CallbackParams callbackParams) {
+        CaseData caseData = callbackParams.getCaseData();
+
+        if (caseData.getRespondent1() != null && !caseData.getRespondent1().getPartyName().isEmpty()) {
+            notificationService.sendMail(
+                caseData.getRespondent1().getPartyEmail(),
+                getTemplate(),
+                addProperties(caseData),
+                getReferenceTemplate(caseData)
+            );
+        }
+        if (caseData.getRespondent2() != null && !caseData.getRespondent2().getPartyName().isEmpty()) {
+            notificationService.sendMail(
+                caseData.getRespondentSolicitor2EmailAddress() != null
+                    ? caseData.getRespondentSolicitor2EmailAddress() :
+                    caseData.getRespondentSolicitor1EmailAddress(),
+                getTemplate(),
+                addPropertiesDef2(caseData),
+                getReferenceTemplate(caseData)
+            );
+        }
+        return AboutToStartOrSubmitCallbackResponse.builder().build();
+    }
+
+    @Override
+    public Map<String, String> addProperties(CaseData caseData) {
+        return Map.of(
+            CLAIM_REFERENCE_NUMBER, caseData.getLegacyCaseReference(),
+            CLAIMANT_V_DEFENDANT, getClaimantVDefendant(caseData),
+            PARTY_NAME, caseData.getRespondent1().getPartyName()
+        );
+    }
+
+    private String getTemplate() {
+        return notificationsProperties.getNotifyUpdateTemplate();
+    }
+
+    private String getReferenceTemplate(CaseData caseData) {
+        return String.format(REFERENCE_TEMPLATE, caseData.getLegacyCaseReference());
+    }
+
+    private String getLegalOrganizationDef2Name(final CaseData caseData) {
+        Optional<Organisation> organisation = organisationService
+            .findOrganisationById(caseData.getApplicant2OrganisationPolicy() != null
+                                      ? caseData.getApplicant2OrganisationPolicy()
+                .getOrganisation().getOrganisationID() : caseData.getApplicant1OrganisationPolicy()
+                .getOrganisation().getOrganisationID());
+        if (organisation.isPresent()) {
+            return organisation.get().getName();
+        }
+        return caseData.getApplicant2().getPartyName();
+    }
+
+    public Map<String, String> addPropertiesDef2(final CaseData caseData) {
+        return new HashMap<>(Map.of(
+            CLAIM_REFERENCE_NUMBER, caseData.getLegacyCaseReference(),
+            CLAIMANT_V_DEFENDANT, getClaimantVDefendant(caseData),
+            PARTY_NAME, getLegalOrganizationDef2Name(caseData)
```

Comment:
for respondent 2 why we need legal org name not the party name ?

Reply:
I changed it to Respondant2 PartyName